### PR TITLE
ci(release): Enable releases through centralized repo

### PR DESCRIPTION
**Pull request by BYK**
Depends on getsentry/action-prepare-release#6

### CODE REVIEW

**Comment by BYK** [comment]
I don't know if this will work properly but worth a shot :)

**Comment by HazAT** [comment]
fyi @st0012 
Please use this action from now on

**Review by HazAT — APPROVED** [review]

**Comment by BYK** [comment]
@st0012 - I hope this aligns with how you do releases. I'll wait for your input before merging.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GH_RELEASE_PAT }}
+          fetch-depth: 0
+      - name: Prepare release
+        uses: getsentry/action-prepare-release@v1.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
+        with:
+          path: sentry-${{ inputs.sdk_name }}
```
Comment (st0012) [inline .github/workflows/release.yml:28]: I think we can just accept `inputs.sdk_name` here. I feel more comfortable to always enter the full gem name.

**Review by st0012 — COMMENTED** [review]
thanks for the suggestion & PR 👍 just left a small suggestion.

File: .github/workflows/release.yml (hunk 1 above)
Comment (BYK) [reply .github/workflows/release.yml:28]: Alright, will update accordingly.

**Comment by st0012** [comment]
@BYK thanks, I will test this flow in the upcoming 4.1.4 release (likely tomorrow) 😄

**Pull request merged**